### ISSUE TITLE
Is es module not supported?

### ISSUE THREAD

**Issue by songyazhao** [electrovir/prettier-plugin-multiline-arrays#30]
My `.prettierrc.js` file：
```js
const plugins = [
  './node_modules/prettier-plugin-multiline-arrays'
];

module.exports = {
  printWidth: 80,
  tabWidth: 2,
  useTabs: false,
  singleQuote: true,
  semi: true,
  bracketSpacing: true,
  trailingComma: 'none',
  arrowParens: 'avoid',
  htmlWhitespaceSensitivity: 'strict',
  vueIndentScriptAndStyle: false,
  endOfLine: 'lf',
  ignorePath: '.prettierignore',
  plugins
};
```
The error is reported as follows：
```bash
ERROR in [eslint] Directory import '/Users/Zero/Project/QingZi/oo-sport/node_modules/prettier-plugin-multiline-arrays' is not supported resolving ES modules imported from /Users/Zero/Project/QingZi/oo-sport/node_modules/prettier/index.mjs
Did you mean to import /Users/Zero/Project/QingZi/oo-sport/node_modules/prettier-plugin-multiline-arrays/dist/index.js?
Occurred while linting /Users/Zero/Project/QingZi/oo-sport/src/config/football.js:6
Rule: "prettier/prettier"

webpack compiled with 1 error
```

**Comment by electrovir** [comment]
Try

```javascript
const plugins = [
  'prettier-plugin-multiline-arrays'
];
```

I noticed that Prettier v3 doesn't work with paths for plugins anymore. I need to update the README of this package to reflect that.

**Comment by AltarBeastiful** [comment]
This fixed an error that was hidden in the prettier console. 
We should probably add this to the main Readme.md ?

**Comment by electrovir** [comment]
Sorry for the delay: fixed README has now been published in [v3.0.3](https://www.npmjs.com/package/prettier-plugin-multiline-arrays/v/3.0.3)
